python/AmcCarrierCore/AppMps/_AppMps.py

```diff
@@ -18,23 +18,24 @@
 #-----------------------------------------------------------------------------
 
 import pyrogue as pr
-import AppMps  as mps
+from AmcCarrierCore.AppMps._AppMpsSalt import AppMpsSalt
+from AmcCarrierCore.AppMps._AppMpsThr  import AppMpsThr
```

Comment:
Python modules that are named with a leading underscore should not be imported like this. Only __init__.py should import them.